### PR TITLE
🐞 Expose the UI in docker

### PR DESCRIPTION
**Pull request by moafred**
Don't know for you, but in docker i needed to set `0.0.0.0` in order to access it.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -19,12 +19,12 @@ services:
     # Please read the https://www.freqtrade.io/en/latest/rest-api/ documentation
     # before enabling this.
     ports:
-      - "127.0.0.1:8080:8080"
+      - "0.0.0.0:8080:8080"
```
Comment (bjwie) [inline docker-compose.yml:22]: "8080:8080"
Would be sufficient as well

**Comment by Rikj000** [comment]
Wasn't an issue on my end, but tried, tested and still working here. So merging it in hoping that it will help others from having the same issues you guys experienced. Thanks! :clap:

**Comment by vbersier** [comment]
Just be careful that now the UI is available on port 8080 from anywhere, make sure your security is on point.

**Comment by Rikj000** [comment]
> Just be careful that now the UI is available on port 8080 from anywhere, make sure your security is on point.

This is true, as always strong keys/passwords are a must!   
But I believe this also was the case before this change in the `docker-compose.yml`

**Comment by bjwie** [comment]
Yes, its correct. I think the majority will run it on a raspberry pi.

**Pull request merged**